### DISCUSSION TITLE
Add RaveSpigot support

### DISCUSSION THREAD

**Discussion by GoodestEnglish** [diamond-rip/Eden#411] [To-do List]
Status: Not started (4/7/2023), Finished (25/7/2023)

Mentioned in https://github.com/diamond-rip/Eden/discussions/102#discussioncomment-6353552

**Comment by HierarchyMan** [comment]
100th discussion 🎉

**Reply by GoodestEnglish** [reply]
Epic

**Reply by GoodestEnglish** [reply]
So I am trying to add RaveSpigot support. But I want to make life easier and not use reflection because it is going to be messy. Do you have ravespigot jar file?

**Reply by HierarchyMan** [reply]
> So I am trying to add RaveSpigot support. But I want to make life easier and not use reflection because it is going to be messy. Do you have ravespigot jar file?

I was actually gonna ask you to help me compile the code to a jar file because the one I have is compiled by a sketchy leaker guy

**Reply by HierarchyMan** [reply]
Heres the link anyways lul: https://drive.google.com/file/d/133xOlvFsVES5WzCc5LiIft6KC3-BfD8k/view?usp=drive_link

**Reply by GoodestEnglish** [reply]
My computer somehow can't compile any maven related project. Only gradle project can be compiled. Thats why :/

**Reply by GoodestEnglish** [reply]
The drive link isn't working :/

**Reply by HierarchyMan** [reply]
https://www.mediafire.com/file/l6un5eaffpxrm69/RaveSpigot.jar/file

**Reply by GoodestEnglish** [reply]
The file you've sent is a RaveSpigot fork, which has different class location compare to the original RaveSpigot. This cannot be used.

**Reply by HierarchyMan** [reply]
> The file you've sent is a RaveSpigot fork, which has different class location compare to the original RaveSpigot. This cannot be used.

So the least you can do is guide me thru compilation process

**Reply by GoodestEnglish** [reply]
Simply put the src into intellij. And compile it.

**Comment by Mixcko** [comment]
> The file you've sent is a RaveSpigot fork, which has different class location compare to the original RaveSpigot. This cannot be used.

Here is the original ravespigot:

https://www.mediafire.com/file/bpuilbrqxwhgm47/RaveSpigot-Server-1.8.8-R0.1-SNAPSHOT.jar/file

**Reply by GoodestEnglish** [reply]
ty for this. Now I can add ravespigot support.

**Reply by Mixcko** [reply]
np
